lib/src/main/java/com/kirkbushman/araw/models/Submission.kt

```diff
@@ -142,6 +142,9 @@ data class Submission(
     @Json(name = "gildings")
     override val gildings: Gildings,
 
+    @Json(name = "sr_detail")
+    val sr_detail: SrDetail?,
```

Comment:
Use camel case for the variable name here, otherwise detekt will error the build in Travis. Move it to srDetail, please.